src/main/java/cms/gov/madie/measure/exceptions/NonUniqueTestCaseName.java

```diff
@@ -0,0 +1,13 @@
+package cms.gov.madie.measure.exceptions;
+
+public class NonUniqueTestCaseName extends RuntimeException {
```

Comment:
Since this is an Exception, should the class name end with Exception e.g. NonUniqueTestCaseNameException?

Reply:
That is should. 

Also, @adongare, do you think I should just name it `DuplicateTestCaseNameException` instead of `NonUnique`?

Reply:
Went ahead with the rename /shrug.

Reply:
Yeah that sounds better